frameworks/hdfs/src/main/dist/ssl-client.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+    <property>
+        <name>ssl.client.truststore.password</name>
+        <value>changeit</value>
+    </property>
+    <property>
+        <name>ssl.client.truststore.type</name>
+        <value>jks</value>
+    </property>
+    <property>
+        <name>ssl.client.truststore.location</name>
```

Comment:
Why add these client configs?  Clients won't have access to this JRE, for example.